src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestEventsConverter.kt

```diff
@@ -261,7 +260,7 @@ class CargoTestEventsConverter(
         private const val NAME_SEPARATOR: String = "::"
         private const val DOCTESTS_SUFFIX = "doctests"
 
-        private val DOCTEST_PATH_RE: Regex = """"[0-9 a-z_A-Z\-\\.]+ - """.toRegex()
+        private val NAME_RE: Regex = """"name":\s+"(?<name>[^"]+)"""".toRegex()
```

Comment:
Spaces are not necessary after `:` in json
```suggestion
        private val NAME_RE: Regex = """"name":\s*"(?<name>[^"]+)"""".toRegex()
```